data/pc/1.12/blockCollisionShapes.json

```diff
@@ -238,7 +238,7 @@
 "lime_glazed_terracotta":1,
 "pink_glazed_terracotta":1,
 "gray_glazed_terracotta":1,
-"silver_glazed_terracotta":1,
+"light_gray_glazed_terracotta":1,
```

Comment:
are you sure about this? they show up as silver_glazed_terracotta/silver_shulker_box in-game.

Reply:
I updated it to be consistent with blocks.json https://github.com/PrismarineJS/minecraft-data/blob/master/data/pc/1.12/blocks.json#L5792 It is also listed as light gray on the wiki https://minecraft.gamepedia.com/Terracotta